lib/openstudio-standards/btap/structure.rb

```diff
@@ -564,18 +542,93 @@ def initialize(model = nil, cat = "commerce", lload = 30)
         mass.setSpace(space)
       end
 
-      # @todo
-      @co2 = 0
+      # Embodied CO2-e kg (A1-A3) of a model is tallied separately as follows:
+      @co2              = {}
+      @co2[:structure ] = 0
+      @co2[:insulation] = 0
+      @co2[:cladding  ] = 0
+
+      # The :structure key/value pair includes above grade 'structures' (e.g.
+      # slabs, columns) and 'framing' (e.g. wood-framed vs steel-framed). Why
+      # grouped together? In many smaller-scale facilities, structure and
+      # framing are synonymous, or at least tightly coupled, e.g.:
+      #   - load-bearing wood-framed walls in small-scale residential
+      #   - load-bearing CMU walls in small-scale industrial
+      #   - metal buildings
+      #
+      # Below-grade structures (rebar + poured concrete) are ignored - no
+      # alternative options are considered for the moment, e.g. lower carbon
+      # concrete mixes.
```

Comment:
Ignoring for the moment:

- foundation walls, footings, slabs (i.e. big embodied carbon numbers)
- cladding, finishes and insulation
- insulation embodied carbon only added once TBD uprates insulation materials (as per NECB 2017/2020)
- thicker insulation will drag in a proportional increase in secondary framing (e.g. double Z-bars)

Until these are added in, reported kg CO2-e/m2 vary wildly! Adding in the above items will:

- increase substantially kg CO2-e/m2 estimations (more in line with published numbers)
- level out observed discrepancies